spec/Result/OkSpec.php

```diff
@@ -100,7 +104,8 @@ function it_unwrapOrs_with_its_value()
     function it_unwrapOrElses_with_its_value()
     {
         $this->beConstructedWith("value");
-        $this->unwrapOrElse(function() {})->shouldBe("value");
+        $this->unwrapOrElse(function () {
+        })->shouldBe("value");
```

Comment:
Especially on places like this. Is it a fixer thing that causes this?

Reply:
Yes. PSR declares newlines for function body.
I think we can skip this rules for specs.

Reply:
I see. I don't mind.